include/libdnf/base/goal.hpp

```diff
@@ -26,6 +26,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "libdnf/rpm/package.hpp"
 
 #include <memory>
+#include <string>
```

Comment:
The line is not needed. And please, remove it from this commit. Commit is about the swig bindings ("Add bindings for Goal class").

Reply:
Sorry I was hunting why swig cannot convert perl string to std::string.